src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.scss

```diff
@@ -0,0 +1,38 @@
+.c-context-menu {
+  display: inline-flex;
+  flex-direction: column;
+  min-width: 180px;
+  max-width: 280px;
+  background-color: rgb(255, 255, 255);
+  user-select: none;
```

Comment:
For it to match the old context menu, it should have a slight border-radius.

Reply:
Added!